00-RELEASENOTES

```diff
@@ -9,6 +9,50 @@ CRITICAL: There is a critical bug affecting MOST USERS. Upgrade ASAP.
 SECURITY: There are security fixes in the release.
 --------------------------------------------------------------------------------
 
+================================================================================
+Valkey 8.1.0 RC2  -  Released Thu 20 March 2025
+================================================================================
+Upgrade urgency LOW: This is the second release candidate of Valkey 8.1, with several bug fixes, 
+control on manual-failover timeout and expended module API to reduce module executed commands overhead.
+
+Performance/Efficiency Improvements - Core
+==========================================
+* Optimize bitcount command by SIMD (#1741)
+* Embed hash value in hash data type entries to reduce memory footprint (#1579)
+
+Cluster modifications
+====================
+* Add cluster-manual-failover-timeout configuration to control the timeout for manual failover (#1690)
+* Improve error message reporting when invalid port is provided for cluster meet command. (#1686)
+* broadcast epoch ASAP when configEpoch changed (#1813)
+
+Module Improvements
+===================
+* Add new module API flag to bypass command validation in order to reduce processing overhead (#1357)
+
+Behavior Changes
+================
+* Enable TCP_NODELAY by default in incoming and outgoing connections (#1763)
+* Move TCP/TLS specific options from generic client to connection type (#1706)
```

Comment:
I think this was a bug fix, not a behavior change. Both this and the next one should move down.